### PR TITLE
[Issue-89] Compressed sparse row solution and learning section

### PR DESCRIPTION
**Pull request by paddywardle**
Added learning and solution section for compressed sparse row matrix format.

### CODE REVIEW

File: Problems/60_compressed_row_sparse_matrix/learn.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+<h3>Structure</h3>
+
+Given a matrix: \[ \begin{bmatrix} 1 & 0 & 0 & 0 \\ 0 & 2 & 0 & 0 \\ 3 & 0 & 4 &
+0 \\ 1 & 0 & 0 & 5 \end{bmatrix} \] The CSR representation would be:
+
+<ul>
+  <li><strong>Values array</strong>: [1, 2, 3, 4, 1, 5]</li>
+  <li><strong>Column indices array</strong>: [0, 1, 0, 2, 0, 3]</li>
+  <li><strong>Row pointer array</strong>: [0, 1, 2, 4, 5]</li>
```
Comment (moe18) [inline Problems/60_compressed_row_sparse_matrix/learn.html:39]: this should be [0, 1, 2, 4, 6]

**Review by moe18 — COMMENTED** [review]
looks good, one issue with the learn section

File: Problems/60_compressed_row_sparse_matrix/learn.html (hunk 1 above)
Comment (moe18) [reply Problems/60_compressed_row_sparse_matrix/learn.html:39]: also add a bit more of an explanation of Row pointer array in the learn section

Comment (paddywardle) [reply Problems/60_compressed_row_sparse_matrix/learn.html]: Updated this :)

**Review by moe18 — APPROVED** [review]

**Pull request merged**